### PR TITLE
update SSL-Config

### PR DESCRIPTION
**Pull request by laclaro**
Updating SSL-config.

### CODE REVIEW

File: SOURCES/privacyidea.conf [hunk 1]
```diff
@@ -26,10 +26,13 @@ WSGIDaemonProcess privacyidea processes=1 threads=15 display-name=%{GROUP} user=
...
-SSLEngine On
-SSLProtocol All -SSLv2 -SSLv3
+
+SSLEngine on
+SSLProtocol ALL -SSLv2 -SSLv3 -TLSv1 -TLSv1.1
 SSLHonorCipherOrder On
-SSLCipherSuite EECDH+AES256:DHE+AES256:EECDH+AES:EDH+AES:-SHA1:EECDH+RC4:EDH+RC4:RC4-SHA:AES256-SHA:!aNULL:!eNULL:!EXP:!LOW:!MD5
+SSLCipherSuite          ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256
+SSLCompression          off
+SSLSessionTickets       off
```
Comment (plettich) [inline SOURCES/privacyidea.conf:35]: Please add a comment why SSL session tickets are turned off (link to https://httpd.apache.org/docs/2.4/mod/mod_ssl.html#sslsessiontickets should suffice).

File: SOURCES/privacyidea.conf [hunk 2]
```diff
@@ -26,10 +26,13 @@ WSGIDaemonProcess privacyidea processes=1 threads=15 display-name=%{GROUP} user=
...
 WSGIPassAuthorization On
 WSGIScriptAlias / /etc/privacyidea/privacyideaapp.wsgi
-SSLEngine On
-SSLProtocol All -SSLv2 -SSLv3
+
+SSLEngine on
+SSLProtocol ALL -SSLv2 -SSLv3 -TLSv1 -TLSv1.1
 SSLHonorCipherOrder On
-SSLCipherSuite EECDH+AES256:DHE+AES256:EECDH+AES:EDH+AES:-SHA1:EECDH+RC4:EDH+RC4:RC4-SHA:AES256-SHA:!aNULL:!eNULL:!EXP:!LOW:!MD5
+SSLCipherSuite          ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256
```
Comment (plettich) [inline SOURCES/privacyidea.conf:33]: This only allows elliptic curve DHE, we should also support regular DHE: https://github.com/ssllabs/research/wiki/SSL-and-TLS-Deployment-Best-Practices#23-use-secure-cipher-suites

**Review by plettich — CHANGES_REQUESTED** [review]
please see my comments.

File: SOURCES/privacyidea.conf (hunk 2 above)
Comment (laclaro) [reply SOURCES/privacyidea.conf:33]: Let's talk about moving the configurations to some central place.

**Pull request closed without merging**